libsoftwarecontainer/unit-test/CMakeLists.txt

```diff
@@ -16,7 +16,7 @@
 #
 # For further information see LICENSE
 
-
+include (AddGTestTest)
```

Comment:
Again inconsistency with the whitespace. Please take a look throughout if there are more.